crates/stackable-versioned-macros/tests/trybuild.rs

```diff
@@ -37,7 +26,6 @@ mod default {
 #[test]
 fn default_macros() {
     let t = trybuild::TestCases::new();
-    t.pass("tests/default/pass/*.rs");
```

Comment:
I'm assuming insta can't snapshots test the failed compilation?

Just checking if trybuild is still useful.

Reply:
I haven't looked into it, but I have a feeling `insta` can't do it. So `trybuild` would still be needed.

Reply:
`insta` currently cannot do it. For now, we keep the (failing) `trybuild` test cases.